src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -556,7 +574,7 @@ function createNotebookWorkspaceResetButton(container: Explorer): CommandButtonC
     onCommandClick: () => container.resetNotebookWorkspace(),
     commandButtonLabel: label,
     hasPopup: false,
-    disabled: false,
+    disabled: useSelectedNode.getState().isQueryCopilotCollectionSelected() ? true : false,
```

Comment:
Same as above, you don't need `? true : false` since `isQueryCopilotCollectionSelected()` should return a boolean anyways.